mmedit/models/editors/inst_colorization/color_utils.py

```diff
@@ -0,0 +1,307 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import numpy as np
+import torch
+
+
+def xyz2rgb(xyz):
+    """Conversion images from lab to xyz.
```

Comment:
Type lint in the function signature is missing. (Maybe can convert to a **good first issue**)

Reply:
Agree.